Sources/DVR/Cassette.swift

```diff
@@ -65,4 +72,15 @@ private extension URLRequest {
 
         return encoded1.isEqual(encoded2)
     }
+
+    func hasHeadersEqualToThatOfRequest(_ request: URLRequest, headersToCheck: [String]) -> Bool {
+        let request1Headers = allHTTPHeaderFields ?? [:]
+        let request2Headers = request.allHTTPHeaderFields ?? [:]
+        for header in headersToCheck {
```

Comment:
Ah, I think I understand. At first I was surprised why in `Session` it was an array of strings. But `headersToCheck` is the list of all headers that must be an exact match.

Reply:
Yes, that is correct.